cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -1197,13 +1201,13 @@ pEpochNoUpdateProp =
       <> Opt.help "The epoch number in which the update proposal is valid."
       )
 
-pGenesisFile :: Parser GenesisFile
```

Comment:
This change seems unnecessary.

Reply:
@Jimbo4350, why do you think so?

This changes allows `pGenesisFile` to be reused in two slightly different contexts.

Reply:
Ah right, didn't catch the other usage.